doc/sphinx/Pacemaker_Explained/options.rst

```diff
@@ -62,6 +62,81 @@ Normally, you will use command-line tools that abstract the XML, so the
 distinction will be unimportant; both properties and options are cluster
 settings you can tweak.
 
+Configuration Value Types
+#########################
+
+Throughout this document, configuration values will be designated as having one
+of the following types:
+
+.. table:: **Configuration Value Types**
+   :class: longtable
+   :widths: 1 3
+
+   +-------------------+-------------------------------------------------------+
+   | Type              | Description                                           |
+   +===================+=======================================================+
+   | boolean           | .. _boolean:                                          |
+   |                   |                                                       |
+   |                   | .. index::                                            |
+   |                   |    pair: type; boolean                                |
+   |                   |                                                       |
+   |                   | Case-insensitive true/false value where "1", "yes",   |
+   |                   | "y", "on", or "true" evaluate as true, and "0", "no", |
+   |                   | "n", "off", "false", or unset evalute as false        |
```

Comment:
I'd put a semicolon after "evaluate as true" since the list itself contains plenty of commas.  And then, the second "evaluate" is spelled wrong.

Reply:
thanks